frontendservice/src/views/specificShop.vue

```diff
@@ -22,6 +22,7 @@ specific language governing permissions and limitations under the License. */
           v-else
           class="title"
         >Number of people currently in queue are {{shopinfo.customersInQueue}}</p>
+        <p v-if="totalWaitingTime != 0" class="title">Your waiting time is: {{totalWaitingTime}} mins</p>
```

Comment:
Nit. Change this to "Approx. waiting time:" ?

Reply:
The variable name should be approxWaitingTime you mean? Or talking about the message displayed in the UI?

Reply:
Changed.